dataservice/api/aliquot/models.py

```diff
@@ -25,7 +25,8 @@ class Aliquot(db.Model, Base):
     analyte_type = db.Column(db.Text(), nullable=False)
     concentration = db.Column(db.Integer())
     volume = db.Column(db.Float())
-    # '2017-06-01'
     shipment_date = db.Column(db.DateTime())
+    samples = db.relationship('Sample', back_populates='aliquots',
```

Comment:
switched to back_populates to delete parent(Sample) when empty child(Aliquot). There by have option to delete orphan reciprocally.